README.md

```diff
@@ -2,36 +2,42 @@
 
 PulseAudio implementation for Home Assistant.
 
-This container ship the upstream ALSA configs and base settings for Pulse.
+This container ships the upstream ALSA configs and base settings for Pulse.
 
 ## How it works
 
-We have a central container they handle the ALSA settings and run on top a PulseAudio service. They service is expose to Home Assistant and Add-ons if they need audio support. This handling allow us to manage all central on one place and streamline the audio as a plaform.
+The central `hassio_audio` container handles the ALSA settings and runs a PulseAudio service on top.
```

Comment:
The `hassio_audio` mention here is, is an internal and thus not user facing. Let's avoid doing that.

Reply:
Would this be better?
```suggestion
The central audio container handles the ALSA settings and runs a PulseAudio service on top.
```